sources/platform/integrations/programming/api.md

```diff
@@ -90,6 +90,25 @@ This is useful if you want to for example create a token that can dynamically cr
 
 :::
 
+### Permission dependencies
+
+Some permissions have dependencies, that means other permissions which need to be granted as well.
+
+Some of these dependencies are enforced already by the form. You will notice that if you for example try to grant the _Write_ permission to a dataset, the _Read_ permission will be automatically selected as well. In other words, you can only write to a dataset if you can also read from it.
+
+![The Write permission depends on Read for a dataset](../images/api-token-scoped-dependencies.png)
+
+However, other dependencies are more complicated, and **it is your responsibility that the token is set up correctly**. Specifically:
+
+- To create or update a Schedule, the token needs access not only to the Schedule itself, but also to the Actor or task that is being scheduled.
+- Similarly, to create or update a task, the token needs the additional permission to access the task's Actor itself.
+
+:::tip
+
+Let's say that you have an Actor and you want to programmatically create schedules for that Actor. Then you can create a token that has the account level _Create_ permission on schedules, but only the resource-specific _Run_ permission on the Actor. Such a token has exactly the permissions it needs, and nothing more.
+
+:::
```

Comment:
```suggestion
#### Manual dependencies

Other dependencies are more complex and require manual configuration. It's your responsibility to set up the token correctly.

Consider the following scenarios:

- Creating or updating a Schedule: The token need access to both the Schedule and the associated Actor or task.
- Creating or updating a task: The token needs permission to access both the task and its Actor.

:::tip

To create a token with minimal permission, consider this example:

You want to programmatically create schedules for an Actor. Create a token with:

- Account-level _Create_ permission on Schedules
- Resource-specific _Run_ permission on the Actor

This token has the permissions it needs, nothing more.

:::
```

Reply:
There is a like nested "colonning" :smile: 

First level: `To create a token with minimal permission, consider this example:`
Second level: `You want to programmatically create schedules for an Actor. Create a token with:`

TBH, the original compact variant looks better to me.